build.sbt

```diff
@@ -105,7 +105,7 @@ def copyAndReplace(srcDirs: Seq[File], destinationDir: File): Seq[File] = {
   // Copy the source files from the base project, exclude classes on java.util and dirs
   val generatedFiles: List[java.io.File] = onlyScalaDirs.foldLeft(Set.empty[File]) { (files, sourceDir) =>
     files ++ copyDirectory(sourceDir, destinationDir, overwrite = true)
-  }.filterNot(_.isDirectory).filterNot(_.getParentFile.getName == "util").toList
+  }.filterNot(_.isDirectory).filter(_.getName.endsWith(".scala")).filterNot(_.getParentFile.getName == "util").toList
```

Comment:
yes, this is exactly for that. It could also be achieved in the `copyDirectory` method by passing a filter that only takes `.scala` files. Something in the likes of `PathFinder(source).**(scalaFileFilter)` but  that may make the build skip copying resources that could be needed afterwards, so I thought that skipping non-scala files before doing the `replaceAll` could suffice.

No problem of rolling this back if you think that the build should not take care of it...